silk/manifest.json

```diff
@@ -0,0 +1,39 @@
+{
+  "display_name": "Silk",
+  "maintainer": "help@datadoghq.com",
+  "manifest_version": "1.0.0",
+  "name": "silk",
+  "metric_prefix": "silk.",
+  "metric_to_check": "silk.system.capacity.free",
+  "creates_events": true,
+  "short_description": "Monitor Silk performance and system stats.",
+  "guid": "75648ebe-8ce3-4b08-bba2-2f957a7e94fa",
+  "support": "core",
+  "supported_os": [
+    "linux",
+    "mac_os",
+    "windows"
+  ],
+  "public_title": "Silk",
+  "categories": [
+    "cloud",
+    "data store",
+    "provisioning"
+  ],
+  "type": "check",
+  "is_public": false,
+  "integration_id": "silk",
+  "assets": {
+    "configuration": {
+      "spec": "assets/configuration/spec.yaml"
+    },
+    "dashboards": {},
```

Comment:
Are you going to be adding a dashboard in this PR? If not, I think you can remove the empty `silk/assets/dashboards/silk_overview.json` file.

Reply:
I'm currently working on a dashboard, although I won't let it be a blocker for merging this PR. I can remove it until I finish the dashboard either before or after this PR is merged.